compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/fold_unit_dims.mlir

```diff
@@ -1,4 +1,5 @@
-// RUN: iree-opt --pass-pipeline="builtin.module(iree-linalg-ext-fold-unit-extent-dims)" %s --split-input-file --mlir-print-local-scope | FileCheck %s
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-linalg-ext-fold-unit-extent-dims{use-reshapes=false})" %s --split-input-file --mlir-print-local-scope | FileCheck %s --check-prefix=SLICE
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-linalg-ext-fold-unit-extent-dims{use-reshapes=true}, canonicalize)" %s --split-input-file --mlir-print-local-scope | FileCheck %s --check-prefix=RESHAPE
```

Comment:
It is okay without using the trick, but I'd like to share the trick for the pass. You can add `CHECK-ALL` to the `--check-prefix` list then you can do something like:

```
// CHECK-ALL-LABEL:  ...
// CHECK-ALL:  %[[INDEX:.+]] = ...
// SLICE:  tensor.extract_slice
// RESHAPE: tensor.collapse_shape
// CHECK-ALL: iree_linalg_ext.scatter
```

Here is an example in IREE:

https://github.com/iree-org/iree/blob/5791cf50c5e140761874a35b53c0a647d94fc359/compiler/src/iree/compiler/Codegen/Common/test/decompose_pack_unpack_ops.mlir#L1-L21